.github/workflows/workflow.yaml

```
@@ -16,7 +16,7 @@ jobs:
       matrix:
         platform:
           - { name: 'Linux',    arch: 'x64',          os: ubuntu-latest,  werror: true }
-          - { name: 'Linux',    arch: 'arm64',        os: ubuntu-latest,  werror: true, cmake-toolchain-file: 'cmake/toolchains/linux-aarch64.cmake', apt-packages: 'gcc-aarch64-linux-gnu g++-aarch64-linux-gnu', cross: true }
+          # - { name: 'Linux',    arch: 'arm64',        os: ubuntu-latest,  werror: true, cmake-toolchain-file: 'cmake/toolchains/linux-aarch64.cmake', apt-packages: 'gcc-aarch64-linux-gnu g++-aarch64-linux-gnu', cross: true }
```

```suggestion
          - { name: 'Linux',    arch: 'arm64',        os: ubuntu-latest,  werror: true, cmake-toolchain-file: 'cmake/toolchains/linux-aarch64.cmake', apt-packages: 'gcc-aarch64-linux-gnu g++-aarch64-linux-gnu', cross: true }
```


I think the issue was with missing pre-installed x11 dependencies on Ubuntu 24.04, which is fixed on main.
Anyhow, if arm64 fails, we should fix this.